docs/src/faq.md

```diff
@@ -0,0 +1,33 @@
+# FAQs
+
+This section collects frequently asked questions about Kani.
+Please consider [opening an issue](https://github.com/model-checking/kani/issues/new/choose) if you have a question that would like to see here.
+
+## Questions
+
+**Question:** Kani doesn't fail after `kani::assume(false)`. Why? [Answer](#assuming-a-false-condition)
```

Comment:
Yes, this is the other option I was talking about. I'm going to write it this way and see if it doesn't limit us in any way (don't know how it will show code snippets).

Thanks for linking to the rendered version earlier!

Reply:
Using HTML tags introduces some problems, but there are workarounds for most of them ([this gist](https://gist.github.com/scmx/eca72d44afee0113ceb0349dd54a84a2) explains how). For example, you cannot use back-ticks on the `summary` content. But you can use `<code>` tags.

Reply:
Interesting... so you basically go into html mode at that point.